UefiCpuPkg/PiSmmCpuDxeSmm/Ia32/SmmProfileArch.c

```diff
@@ -72,3 +72,15 @@ ClearTrapFlag (
 {
   SystemContext.SystemContextIa32->Eflags &= (UINTN) ~BIT8;
 }
+
+/**
+  Create new entry in page table for page fault address in SmmProfilePFHandler.
+
+**/
+VOID
+SmiProfileMapPFAddress (

```

Comment:
SmiProfileXXX is not good since it's another feature not same to the SmmProfile.

how about change to SmmProfileMapPFAddress?

Reply:
Changed to SmmProfileMapPFAddress

Reply:
ok